JS/Lesson13/sinan/script.js

```diff
@@ -2,143 +2,177 @@
 // Declare a variable y and initialize it to 3.
 // Declare a variable z and initialize it to the sum of x and y.
 // var, const, let
+
 const x = 5;
 const y = 3;
-const z = x + y;
-console.log('Exercise 1:', z);
+const z = x+y;
+console.log('Exercises1 :', z);
 
-console.log('------------------------');
-const name = "onur";
-const age = 32;
-const phrase = "Hello world!";
-const myAge = `${phrase} My name is ${name}. I'm ${age} years old.`;
-const myAge2 = phrase + " My name is " + name + ". I'm " + age + " years old.";
 
-console.log('Exercise 2:',myAge);
-console.log('Exercise 2.2:',myAge2);
+console.log('------------------------');
 
 // ---
 // 2. Declare a variable name and initialize it to your name as a string.
 // Declare a variable age and initialize it to your age as a number.
 // Declare a variable phrase and initialize it to the string "Hello, world!".
 // Add name and age to the sentence above. "Hello, world! My name is (name). I'm (age) years old.".
 
+
+const myName='sinan';
+const myAge= 28;
+const phrase ='Hello, world!';
+const salute = '${phrase} My name is ${myName} Im ${myAge} years old.';
```

Comment:
If you want to use string interpolation in JS, you need to use '`', not normal quotes "/'

Reply:
```suggestion
const salute = `${phrase} My name is ${myName} Im ${myAge} years old.`;
```

Reply:
Hi @daglisinan, you still need to fix this